extensions/wikia/TemplateDraft/TemplateConverter.class.php

```diff
@@ -111,6 +111,27 @@ public function createDataTag( $variable ) {
 	}
 
 	/**
+	 * Overrides content of parent page with contents of draft page
+	 * @param Title $title Title object of sub page (draft)
+	 * @throws PermissionsException
+	 */
+	public function approveDraft( Title $title ) {
+		if ( $title->userCan( 'edit' ) ) {
+			// Get contents of draft page
+			$article = Article::newFromId( $title->getArticleID() );
+			$draftContent = $article->getContent();
+			// Get parent page
+			$parentTitleText = $title->getBaseText();
+			$parentTitle = Title::newFromText( $parentTitleText, $title->getNamespace() );
+			$page = WikiPage::newFromID( $parentTitle->getArticleID() );
```

Comment:
Now I need instance of parent title to check edit permissions anyway, so I don't think getParentTitleId can simplify something.